### ISSUE TITLE
Loss is nan, stopping training

### ISSUE THREAD

**Issue by xiaopanchen** [yqc123456/HKD_for_person_search#6]
Hi, when I train the model COAT-HKD on PRW dataset, there is an error "Loss is nan, stopping training". And, I have try reduce the learning rate from 0.003 to 0.0000005. But this error still occurs. We find that the values of some loss are nan. The logs are as following.  We run the train.py on Windows paltform. Thk.

Epoch: [0]  [ 600/2852]  eta: 0:18:03  lr: 0.000000  loss: 37.0373 (40.1406)  loss_rcnn_cls_1st: 1.1283 (1.1437)  loss_rcnn_reg_1st: 0.8622 (0.9942)  loss_rcnn_cls_2nd: 0.7887 (0.7908)  loss_rcnn_reg_2nd: 0.2760 (0.3708)  loss_rcnn_cls_3rd: 0.7877 (0.7958)  loss_rcnn_reg_3rd: 0.0237 (0.0317)  loss_rcnn_reid_2nd: 3.7497 (4.8222)  loss_rcnn_reid_3rd: 4.0176 (4.9138)  oim_loss3: 1.2678 (3.0437)  kd_logits3: 9.7725 (10.0305)  kd_sim_kl3: 2.8775 (3.2199)  loss_rpn_reg: 0.0177 (0.0238)  loss_rpn_cls: 0.6923 (0.6921)  loss_ce_2nd: 3.0903 (3.0894)  loss_ce_3rd: 3.0890 (3.0891)  loss_ce_3rd_teac: 3.0891 (3.0893)  time: 0.4658  data: 0.0004  max mem: 14301
Loss is nan, stopping training
{'loss_rcnn_cls_1st': tensor(1.1173, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_reg_1st': tensor(1.2229, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_cls_2nd': tensor(0.7933, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_reg_2nd': tensor(0.0719, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_cls_3rd': tensor(0.7894, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_reg_3rd': tensor(0.0001, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_reid_2nd': tensor(nan, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rcnn_reid_3rd': tensor(nan, device='cuda:0', grad_fn=<MulBackward0>), 'oim_loss3': tensor(nan, device='cuda:0', grad_fn=<MulBackward0>), 'kd_logits3': tensor(7.4464, device='cuda:0', grad_fn=<MulBackward0>), 'kd_sim_kl3': tensor(0.1696, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rpn_reg': tensor(0.0030, device='cuda:0', grad_fn=<MulBackward0>), 'loss_rpn_cls': tensor(0.6915, device='cuda:0', grad_fn=<MulBackward0>), 'loss_ce_2nd': tensor(nan, device='cuda:0', grad_fn=<MulBackward0>), 'loss_ce_3rd': tensor(nan, device='cuda:0', grad_fn=<MulBackward0>), 'loss_ce_3rd_teac': tensor(nan, device='cuda:0', grad_fn=<MulBackward0>)}

Process finished with exit code 1
